### DISCUSSION TITLE
Interface problem

### DISCUSSION THREAD

**Discussion by facundogithub** [Stellarium/stellarium#2294] [Q&A]
Hi, I recently installed Stellarium on a new laptop.
But a few seconds after running the app I get these errors on the screen and I can't see it normally.

![20220220_164526](https://user-images.githubusercontent.com/100150762/154992440-e6b40986-260c-46f5-b5d5-5d62603bc420.jpg)

I tried all the versions and none of them worked.
It is a laptop with windows 11, core i7 gen 11, 16GB.
DirecX 12.

Do you know how I can correct it?

Best regards.

**Comment by gzotti** [comment]
We have seen a few such reports. The FAQ suggest to 
- update graphics drivers
- Use ANGLE mode

**Reply by facundogithub** [reply]
I have all drivers updated y try Angle Mode, but i had the same problem.
thx for the answer
